lib/repositories/survey_repository.dart

```diff
@@ -28,13 +29,7 @@ class SurveyRepositoryImpl implements SurveyRepository {
         final surveysList = <Survey>[];
 
         surveysResponse.edges.forEach((item) {
```

Comment:
What do you think about using `map` here? 🤔

Reply:
oh yes, I forgot at that time to use map, then I will refactor it now and remove the interim `resultList`

Reply:
updated 🙋